runtime/test/test_argument_conversion.cpp

```diff
@@ -146,7 +192,7 @@ void test_scalars(mlir::MLIRContext *ctx) {
   // clang-format on
 
   {
-    std::string x = "Hi, there!";
```

Comment:
Does string not work at all now?

Reply:
I would not but we currently don't allow strings to be passed as arguments.